plugins/crypto-kms/src/test/java/org/opensearch/crypto/kms/KmsServiceTests.java

```diff
@@ -131,10 +131,6 @@ public void testClientSettingsReInit() {
                         assertTrue(credentials instanceof AwsBasicCredentials);
                     }
 
-                    assertEquals(
```

Comment:
Note we test the constituent components of `proxyConfiguration` below. We should not expect the `toString()` output to be stable so I don't think we should have this assertion in the unit tests.

Reply:
Thanks Andrew 👍